lib/Projects/ProjectFactory.php

```diff
@@ -27,10 +28,8 @@ public function create(array $project) : Project
 
         $projectJson = $this->projectJsonReader->read($project['repositoryName']);
 
-        if ($projectJson !== null) {
-            $project = $project + $projectJson;
-        }
+        $projectData = array_merge($projectJson, $project);
```

Comment:
as you deal with associative arrays, why not `array_replace` instead ?

Reply:
Done.